man/sd_notify.xml

```diff
@@ -392,6 +425,21 @@
 
       <programlisting>sd_pid_notify_with_fds(0, 0, "FDSTORE=1\nFDNAME=foobar", &amp;fd, 1);</programlisting>
     </example>
+
+    <example>
+      <title>Eliminating race conditions</title>
+
+      <para>When the client sending the notifications is not spawned
+      by the service manager, it may exit too quickly and the service
+      manager may fail to attribute them correctly to the unit. To
+      prevent such races, use sd_notify_barrier() to syncrhonize against
```

Comment:
enclose the function name in `<function>` please, how we usually to it.

also: syncrhonize is a typo